src/test/FreeC/Environment/RenamerTests.hs

```diff
@@ -56,39 +53,47 @@ genVernacularCommand :: Gen String
 genVernacularCommand = oneof $ map return vernacularCommands
 
 -------------------------------------------------------------------------------
--- Tests for @mustRenameIdent@                                              --
+-- Tests for @mustRenameIdent@                                               --
 -------------------------------------------------------------------------------
-
 -- | Test group for 'mustRenameIdent' tests.
 testMustRenameIdent :: Spec
-testMustRenameIdent = describe "mustRenameIdent" $ do
-  it "keywords must be renamed" $ do
-    property $ forAll genKeyword $ flip mustRenameIdent emptyEnv
-  it "reserved identifiers must be renamed" $ do
-    property $ forAll genReservedIdent $ flip mustRenameIdent emptyEnv
-  it "defined identifiers must be renamed" $ do
-    property $ forAll genIdent $ \ident ->
-      let env = addEntry
-            TypeVarEntry { entrySrcSpan   = NoSrcSpan
-                         , entryName      = IR.UnQual (IR.Ident ident)
-                         , entryIdent     = Coq.bare ident
-                         , entryAgdaIdent = undefined -- ignore Agda identifiers for the moment - TODO: add Unit Tests for Agda renamer!
-                         }
-            emptyEnv
-      in  mustRenameIdent ident env
+testMustRenameIdent = describe "mustRenameIdent"
+  $ do
+    it "keywords must be renamed"
+      $ do
+        property $ forAll genKeyword $ flip mustRenameIdent emptyEnv
+    it "reserved identifiers must be renamed"
+      $ do
+        property $ forAll genReservedIdent $ flip mustRenameIdent emptyEnv
+    it "defined identifiers must be renamed"
+      $ do
+        property
+          $ forAll genIdent
+          $ \ident ->
+          let env = addEntry TypeVarEntry
+                { entrySrcSpan   = NoSrcSpan
+                , entryName      = IR.UnQual (IR.Ident ident)
+                , entryIdent     = Coq.bare ident
+                , entryAgdaIdent = undefined -- ignore Agda identifiers for the moment - TODO: add Unit Tests for Agda renamer!
```

Comment:
This line is 109 characters long with indentation (93 without), so Floskell doesn't appear to split up trailing comments that are too long.

Reply:
```suggestion
                , entryAgdaIdent = undefined -- TODO add Agda renamer tests.
```